tel2puml/tel2puml_types.py

```diff
@@ -40,6 +40,18 @@ class PVEvent(TypedDict):
     eventType: str
 
 
+class PVEventModel(BaseModel):
```

Comment:
can you do some pre validation to make sure for the field previousEventIds that if we get a string we turn it into a list of strings

Reply:
this will also need testing